packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -850,9 +855,65 @@ watch(
 onUnmounted(() => {
 	kernelManager.shutdown();
 });
+
+// Format dates as a readable string
+function formatTimestamp(timestamp) {
+	const date = new Date(timestamp);
+
+	const formatter = new Intl.DateTimeFormat('en-US', {
+		year: 'numeric',
+		month: 'long',
+		day: '2-digit',
+		hour: '2-digit',
+		minute: '2-digit',
+		second: '2-digit',
+		timeZoneName: 'short'
+	});
+
+	return formatter.format(date);
+}
```

Comment:
We already have several different date-formatters under `src/utils/date.ts`, can we just reuse one of the existing ones? If not, this function should be included into `src/util/dates.ts` rather than as a stand-alone in the component.

Reply:
makes sense. I moved the function to `src/util/dates.ts` as suggested.